src/sorcha/ephemeris/simulation_driver.py

```diff
@@ -70,98 +74,59 @@ def create_ephemeris(orbits_df, pointings_df, args, configs):
     for _, pointing in pointings_df.iterrows():
         mjd_tai = float(pointing["observationStartMJD"])
         ra, dec = float(pointing["fieldRA"]), float(pointing["fieldDec"])
-        visit_vec = ra_dec2vec(ra, dec)
```

Comment:
This is the large block of code that was called out in issue #622 as something that could be pre-computed.